src/main/java/com/iota/iri/service/tipselection/impl/WalkValidatorImpl.java

```diff
@@ -75,19 +82,30 @@ public boolean isValid(Hash transactionHash) throws Exception {
         return true;
     }
 
-    private boolean belowMaxDepth(Hash tip, int depth) throws Exception {
+    private boolean belowMaxDepth(Hash tip, int allowedSnapshotIndex) throws Exception {
```

Comment:
could we call this `minimumAllowedSnapshotIndex` or `lower...`?

Reply:
lower